src/lib/pdf_c_bindings.cc

```diff
@@ -49,8 +49,6 @@
  * - \b web.enableIntelligentShrinking Should we enable intelligent shrinkng to fit more content
  *      on one page? Must be either "true" or "false". Has no effect for wkhtmltoimage.
  * - \b web.minimumFontSize The minimum font size allowed. E.g. "9"
- * - \b web.printMediaType Should the content be printed using the print media type instead
```

Comment:
This is technically a breaking change in the API. While you've fixed it for the CLI users, do you think it'll be possible to add backward compatibility for the API users as well somehow?

Reply:
No, unfortunately it seems that the two blocks of settings are treated as independent entities.

Reply:
If you keep it, and then set it to print if either flag is set -- it should work for existing API users, if I'm not mistaken. Is it possible to try that out?

Reply:
`MultiPageLoader::addResource` receives (and passes on) only `settings::LoadPage` but not `settings::Web`.

Is it a sane idea to patch `PdfConverterPrivate::beginConvert` with something like `s.load.printMediaType |= s.web.printMediaType`?

Reply:
That won't work because you may have multiple objects involved. The simplest approach seems to be to modify `wkhtmltopdf_set_object_setting` as follows:
```c
if (!strcmp(name, "web.printMediaType")) {
  name = "load.printMediaType";
}
```
so that this gets carried forward everywhere else.

Reply:
Done